opm/material/fluidsystems/BlackOilFluidSystem.hpp

```diff
@@ -1290,75 +1324,58 @@ class BlackOilFluidSystem : public BaseFluidSystem<Scalar, BlackOilFluidSystem<S
     static bool isInitialized_;
 };
 
-template <class Scalar>
-const int BlackOilFluidSystem<Scalar>::phaseToSolventCompIdx_[3] =
```

Comment:
What are you replacing this with? The associated functions are still present, it seems to me?

Reply:
see the associated functions: https://github.com/OPM/opm-material/pull/325/files#diff-34445ab2480f952e40bee70ce9328c40R451

Reply:
I see, that change in itself is very good. It was precisely those functions that were causing trouble in my previous testing, with this it should be bullet-proof (or at least bullet-proofer (yay, new word)).